modules/peers.js

```diff
@@ -813,11 +813,15 @@ Peers.prototype.onPeersReady = function() {
 				updatePeers(seriesCb) {
 					let updated = 0;
 					const peers = library.logic.peers.list();
+					library.logger.trace('Total list of peers', { count: peers.length });
 
-					library.logger.trace('Updating peers', { count: peers.length });
+					const acceptablePeers = self.acceptable(peers);
```

Comment:
I'm not sure why we need to filter with acceptable here.
When we discover, it is already filtered with this function, and it should be already only acceptable in the list